src/views/Profile/AuthorsList/AuthorListDropdown.tsx

```diff
@@ -2,10 +2,12 @@ import React from 'react';
 import { FormControl, Select, InputBase, MenuItem } from '@material-ui/core';
 import { useStyles } from './styles/AuthorListDropdown.styles';
 
+export type PageSizeType = Array<number | 'All'>;
+
 interface IAuthorListDropdownProps {
-  pageSizes: number[];
-  selected: number;
-  setChanges: (payload: any) => void;
+  pageSizes: PageSizeType;
+  selected: number | string | undefined;
```

Comment:
but what if we had `type PageSizeType = number | 'All'`? Then we could type `selected: PageSizeType | undefined`

or does it create any issues in other code?